### PR TITLE
Added Ability to Test Arion

### PR DESCRIPTION
**Pull request by zzxgzgz**
What this PR does:

1. Added ability to test with Arion Master and Arion DP.
2. Minor Improvements such as adding ability to execute pings on remote machines.
3. Added ability to test multiple compute nodes.
4. Added option to include Arion Agent to the test workflow.

Usage:
Please refer to [docs/test/e2eTestWithArionSetup.adoc](https://github.com/futurewei-cloud/alcor/pull/749/files#diff-a1569ecb49ba815d50ec802ab66a599479217ef4d0384fb7b50a31eb84972b59)

Attaching an example output for future reference:
[test_controller_with_arion.log](https://github.com/futurewei-cloud/alcor/files/9610181/test_controller_with_arion.log)

### CODE REVIEW

**Review by lfu-ps — APPROVED** [review]
lgtm

File: services/pseudo_controller/src/main/resources/application.properties [hunk 1]
```diff
@@ -13,14 +13,24 @@
...
+### otherwise, use arion agent to program ebpf maps on wings.
+use_arion_agent = false
 ### Test Controller NCM test Params ###
 number_of_vpcs = 10
 ports_node_one = 100
 ports_node_two = 200
 node_one_ip = 127.0.0.1
+node_one_mac = bb:ee:dd:00:00:01
 node_two_ip = 127.0.0.2
+node_two_mac = bb:ee:dd:00:00:02
```
Comment (xieus) [inline services/pseudo_controller/src/main/resources/application.properties:27]: What nodes do "node_one_ip" and "node_two_ip" refer to? maybe we can have a better name to differentiate with compute nodes.

File: services/pseudo_controller/src/main/resources/arion_data.json [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+      "description": "znode90",
+      "name": "znode90",
+      "ip_control": "192.168.20.190",
+      "id_control": "ubuntu",
+      "pwd_control": "Lab12345",
+      "inf_tenant": "ens4",
+      "mac_tenant": "52:54:00:5e:99:ac",
+      "inf_zgc": "ens3",
+      "mac_zgc": "52:54:00:f5:aa:52",
+      "arion_agent_location": "absolute_path_of_arion_agent_binary"
```
Comment (xieus) [inline services/pseudo_controller/src/main/resources/arion_data.json:22]: Do we have a document to describe the meaning of each field? I remember Zeta has a doc which can be referenced here.

**Review by xieus — APPROVED** [review]
LGTM

File: services/pseudo_controller/src/main/resources/application.properties (hunk 1 above)
Comment (xieus) [reply services/pseudo_controller/src/main/resources/application.properties:27]: @zzxgzgz

Comment (zzxgzgz) [reply services/pseudo_controller/src/main/resources/application.properties:27]: These are fields used when Test Controller was only able to set up 2 compute nodes, they shouldn't be used now and we can remove them.

File: services/pseudo_controller/src/main/resources/arion_data.json (hunk 2 above)
Comment (zzxgzgz) [reply services/pseudo_controller/src/main/resources/arion_data.json:22]: hmm, I couldn't find the doc you are referring to. But The `NODE_data` refers to information for the wings.

Comment (xieus) [reply services/pseudo_controller/src/main/resources/arion_data.json:22]: Could we provide a quick doc in a separate PR to detail the meaning of each field? Thanks. @zzxgzgz

Comment (zzxgzgz) [reply services/pseudo_controller/src/main/resources/arion_data.json:22]: Sure, we can do that.

File: services/pseudo_controller/src/main/resources/application.properties (hunk 1 above)
Comment (zzxgzgz) [reply services/pseudo_controller/src/main/resources/application.properties]: Removed legacy variables in [f7fa684](https://github.com/futurewei-cloud/alcor/pull/749/commits/f7fa684d19769e17279be7a85d644be34dbdc6ad)

**Pull request merged**